tests/test_markitdown.py

```diff
@@ -7,6 +7,7 @@
 import requests
 
 from markitdown import MarkItDown
+from pathlib import Path
```

Comment:
Are we using Path elsewhere in this tests file?

Otherwise, PR looks good.

Reply:
no usage of it

Reply:
removed it thank you